### DISCUSSION TITLE
Interactive Plotting

### DISCUSSION THREAD

**Discussion by alegresor** [QMCSoftware/QMCSoftware#257] [Ideas]
[Plotly](https://plotly.com/) may be a good way to go for interactive plots in python. 
There also seems to be a nice [Plotly Wordpress plugin](https://wordpress.org/plugins/wp-plotly/) that could enhance our blog posts.

**Comment by fjhickernell** [comment]
Thanks Aleksei,

Can you give us a brief tutorial?

Best,
Fred


Fred J. Hickernell
Vice Provost for Research, research.iit.edu <https://research.iit.edu/>
www.iit.edu/research/about/magazine <https://www.iit.edu/research/about/magazine>
Professor, Department of Applied Mathematics
Center for Interdisciplinary Scientific Computation (CISC) cos.iit.edu/cisc/ <https://cos.iit.edu/cisc/>
Illinois Institute of Technology
IIT Tower, Room 7C9-2, 10 W 35th St, Chicago, IL 60616
***@***.*** ***@***.***>, +1 312 567 3470 <tel:+1%20312%20567%203470>  

Working remotely.  Please contact me by email or phone.

**Comment by alegresor** [comment]
Sure! Let me look more into it before showing the group.

- Aleksei
